package.json

```diff
@@ -48,7 +48,7 @@
   "main": "webpack.config.js",
   "keywords": [],
   "dependencies": {
-    "identity-style-guide": "^0.2.4",
+    "identity-style-guide": "18f/identity-style-guide#bh-add-app-styles",
```

Comment:
@gemfarmer Just updated with new npm version number, can you take a look and 👍  if you are good with it?